### PR TITLE
add webhook processing for merge_group

### PR DESCRIPTION
**Pull request by yjbanov**
Adds handling of the `merge_group` Github request. The handling only logs the request. It does not kick off any CI jobs just yet.

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

File: app_dart/lib/src/request_handlers/github/webhook_subscription.dart [hunk 1]
```diff
@@ -211,6 +214,49 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     }
   }
 
+  /// Handles a GitHub webhook with the event type "merge_group".
+  ///
+  /// Unlike a pull request a merge group contains multiple commits created
+  /// from multiple squashed pull requests. A merge group is therefore not
```
Comment (jtmcdole) [inline app_dart/lib/src/request_handlers/github/webhook_subscription.dart:220]: is this part true? the "branch" is actually a squash of all PRs at that point?  My understanding was there was a merge_group created and checks requested for each PR in the merge queue; with CI running for each request in the merge queue.

Comment (yjbanov) [reply app_dart/lib/src/request_handlers/github/webhook_subscription.dart:220]: I'll reword it. Each PR is squashed individually and produces one commit, then that commit is stacked on top of other commits in the merge group. So a group will contain multiple commits, one per PR. There's no extra squashing done.

Comment (jtmcdole) [reply app_dart/lib/src/request_handlers/github/webhook_subscription.dart]: But we fire of CI for each PR?

File: test_utilities/bin/prepare_environment.sh [hunk 2]
```diff
@@ -9,5 +9,5 @@
 set -ex
 
 dart pub global activate tuneup
-flutter channel master
+flutter channel stable
```
Comment (jtmcdole) [inline test_utilities/bin/prepare_environment.sh:12]: intentional?

**Review by jtmcdole — APPROVED** [review]

File: test_utilities/bin/prepare_environment.sh (hunk 2 above)
Comment (yjbanov) [reply test_utilities/bin/prepare_environment.sh]: Was trying to see if stable `dart analyze` would be less crashy, but it crashed in another way, so I reverted it.

**Pull request merged**